.solhintignore

```diff
@@ -1,5 +1,7 @@
 contracts/Migrations.sol
 .solhint.json
 .solhintignore
+.soliumignore
```

Comment:
unrelated changes. Can you base this diff off the one that introduced those to make reviewing easier?

Reply:
This branch assumes whitespace2 has been merged.

Reply:
Sure, in the future you could base your PR on the one that it assumes will have been merged before. Otherwise it's really hard for your reviewers to see which changes are relevant to this PR.

Reply:
The problem I see with that is that I would either
1. have to look at those hideous lint errors in my editor while working on this (which I am not willing to do) or 
2. wait until the other branch is merged before submitting this at all. 

Let hope that there are no more gross whitespace problems in the future and this will no longer be a concern.